src/svgcontext.ts

```diff
@@ -2,40 +2,65 @@
 // @author Gregory Ristow (2015)
 
 import { Vex } from './vex';
+import { RenderContext } from './types/common';
 
-const attrNamesToIgnoreMap = {
+const attrNamesToIgnoreMap: { [key: string]: any } = {
```

Comment:
There are quite a few `any` without
// eslint-disable-next-line
If they cannot be resolved I suggest to include the comments

Reply:
The type here can be `Attributes` from below (and then you won't need the `eslint-disable`.)